EnglishFilter/sections/foreign.txt

```diff
@@ -4780,6 +4780,8 @@ fxp.co.il#@#iframe[width="300"][height="250"]
 !------------------------------ Italian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78254
+ideegreen.it#?#div[class*="widget_text"]:has(> div.textwidget > center:contains(Advertisements))
```

Comment:
`ideegreen.it#?##sidebar > div.widget_text:has(> div.textwidget > center > font > i:contains(Advertisements))`

Reply:
It does the same thing so what's wrong?

Reply:
Your rule is not accurate compared to the above.